katsdptelstate/test/test_rdb_handling.py

```diff
@@ -143,6 +146,28 @@ def test_zset_with_big_entry(self):
     #     """Ziplist with >4GB of data (can't be encoded as ziplist)"""
     #     self._test_zset([(b'%03d' % i) + b'?' * 500000000 for i in range(10)])
 
+    def _test_hash(self, items):
+        self.tr.hmset('my_hash', items)
```

Comment:
HMSET is deprecated in 4.0.0

Reply:
Unfortunately redis-py hasn't implemented HSET with multiple key-value pairs so we're stuck with it for now.

Reply:
Sure, but you could just call HSET directly or even just multiple times :)

Reply:
I feel like we should burn that bridge when we get to it. Hopefully redis-py will add support (and may even map the hmset method to an HSET redis call) before we get bitten by this.